pkg/console/operator/status.go

```diff
@@ -55,6 +55,7 @@ const (
 	reasonSyncLoopProgressing = "SyncLoopProgressing"
 	reasonNoPodsAvailable     = "NoPodsAvailable"
 	reasonSyncError           = "SyncError"
+	reasonAsExpected          = "AsExpected"
```

Comment:
This `reason` is also used by [kube-apiserver-operator](https://github.com/openshift/cluster-kube-apiserver-operator)

Reply:
hmm so I don't see this reason is used anywhere :-/

Reply:
I think since I removed the manual handling of `status` for the override & instead am using `NewUnsupportedConfigOverridesController()`, its now unused.  However, it should be here and we should use it for statuses if all is well.

Reply:
I'd prefer not to restart the test process over that, might be another 2 weeks till we pass 😄 😭